alf/trainers/evaluator.py

```diff
@@ -170,7 +172,8 @@ def _worker(job_queue: mp.Queue,
             pre_configs: Dict,
             num_parallel_envs: int,
             root_dir: str,
-            seed: Optional[int] = None):
+            seed: Optional[int] = None,
+            create_eval_env: Optional[bool] = False):
```

Comment:
this argument seems unnecessary. It's always False

Reply:
Removed this argument